RELEASE_NOTES.md

```diff
@@ -463,11 +467,17 @@ The following is a summary of the main features added to darktable
 
 ### Other Lua changes
 
-- Add image orientation retrieval support.
+- Added aspect_ratio field to dt_lua_image_t for image orientation
+  retrieval support.
 
 - Two new properties have been added to get the flags (category,
   private) and the synonyms from a tag.
 
+- Moved `pixelpipe-processing-complete` event from the end of the 
```

Comment:
This is the PR you've closed, so if I'm not mistaken it is not in master.

Reply:
I mistakenly removed the branch when I was cleaning stuff out.  It's back now.  Sorry about the confusion.